wfe2/verify.go

```diff
@@ -292,11 +294,17 @@ func (wfe *WebFrontEndImpl) matchJWSURLs(outer, inner *jose.JSONWebSignature) *p
 	return nil
 }
 
+// bJSONWebSignature wraps a *jose.JSONWebSignature, but ensures it has been
+// created and processed by the various safety/sanity checks in wfe.parseJWS.
```

Comment:
Let's beef up this comment a bit. In particular, let's make sure to call out that no one should ever construct a bJSONWebSignature on their own; the only "correct" way to get one is from `parseJWS` or `parseJWSRequest`.